distribution/docs/src/main/resources/content/_managing/_configuring/configuring-multi-factor-auth.adoc

```diff
@@ -0,0 +1,24 @@
+:title: Configuring Multi Factor Authentication
+:type: subConfiguration
+:status: published
+:parent: Configuring REST Services for Users
+:summary: Configuring Multi Factor Authentication using Keycloak as an IdP.
+:order: 23
+
+// This is for MFA
+== {title}
+
+Mutli-factor authentication, sometimes referred to as two-factor authentication, allows for greater security.
+It does this by requiring users to provide multiple proofs of identity, typically through something they know, and something they have/are.
+The IdP that comes with ${branding} does not support multi-factor authentication by default.
+
+Keycloak can be used to help setup and configure multi-factor authentication.
+See <<{managing-prefix}connecting_to_an_external_identity_provider,Connecting to an External Identity Provider>> on how to initially hookup Keycloak.
+
+.Configuring Keycloak for MFA
+. Download and install Keycloak from here: https://www.keycloak.org/downloads.html[Keycloak Downloads^] {external link}
+. Choose an operation mode and configure it based off this information here: https://www.keycloak.org/docs/latest/server_installation/index.html#_operating-mode[Server Installation^] {external link}
```

Comment:
maybe use `see https://www.keycloak.org/docs/latest/server_installation/index.html#_operating-mode[Keycloak Server Installation Documentation^] {external link} to choose an operation mode.`